src/Aquifer.Jobs/Subscribers/TranslationSubscriber.cs

```diff
@@ -0,0 +1,383 @@
+        {
+            translatedResourceContentVersion.ResourceContent.Status = ResourceContentStatus.TranslationEditorReview;
+            await _resourceHistoryService.AddStatusHistoryAsync(
+                translatedResourceContentVersion,
+                ResourceContentStatus.TranslationEditorReview,
+                changedByUserId: message.StartedByUserId,
+                ct
+            );
+
+            await _dbContext.SaveChangesAsync(ct);
+        }
+    }
+
+    /// <summary>
+    /// Translates a project's resources and then takes a final action on the project itself.
+    /// This is done using the fan out/fan in pattern
+    /// (see https://learn.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-overview#fan-in-out)
+    /// which requires using durable functions to maintain state.
+    /// (see https://learn.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-bindings#durableTaskClient-usage).
+    /// </summary>
+    [Function(nameof(TranslateProjectResources))]
+    public async Task TranslateProjectResources(
+        [QueueTrigger(Queues.TranslateProjectResources)] QueueMessage queueMessage,
+        [DurableClient] DurableTaskClient durableTaskClient,
+        CancellationToken ct)
+    {
+        var message = queueMessage.Deserialize<TranslateProjectResourcesMessage, TranslationSubscriber>(_logger);
+
+        var projectResourceContentIds = await _dbContext.Projects
+            .Where(p => p.Id == message.ProjectId)
+            .SelectMany(p => p.ProjectResourceContents.Select(prc => prc.ResourceContentId))
+            .ToListAsync(ct);
+
+        if (projectResourceContentIds.Count == 0)
+        {
+            throw new InvalidOperationException($"Project ID {message.ProjectId} does not have any resource contents.");
+        }
+
+        // Kick off the durable function orchestration.
+        // If it fails it will put a message on the poison queue.
+        await durableTaskClient.ScheduleNewOrchestrationInstanceAsync(
+            nameof(OrchestrateProjectResourcesTranslation),
+            new OrchestrateProjectResourcesTranslationDto(
+                message.ProjectId,
+                message.StartedByUserId,
+                projectResourceContentIds,
+                Queues.GetPoisonQueueName(Queues.TranslateProjectResources),
+                queueMessage.MessageText),
+            ct);
+
+        _logger.LogInformation(
+            "Kicked off {FunctionName} for Project ID {ProjectId}.",
+            nameof(OrchestrateProjectResourcesTranslation),
+            message.ProjectId);
+    }
+
+    /// <summary>
+    /// Durable orchestration function.
+    /// This function is single-threaded and thus should not do any DB operations or any substantial processing.
+    /// It should only orchestrate activities.
+    /// </summary>
+    [Function(nameof(OrchestrateProjectResourcesTranslation))]
+    public async Task OrchestrateProjectResourcesTranslation(
```

Comment:
1. We only need one queue for project translations instead of three (project start, resource translate, project post-translate).
2. Each of the durable function activities builds in its own retry logic and error handling without me needing to do any coding.
3. And the biggest reason: fan out -> fan in.  The project orchestration triggers the translation of each resource in its own function and then the final project post translation function is run _only once_ after all resources have been translated, all with built in retry and error handling.  We had a proposed solution to try to solve that same concern with just queues but it got tricky because we would have needed the post-project translate queue subscriber to run after every resource translation to see if all resources had been translated in order to do the final work for the project.  That solution also came with potential race conditions and the danger of multiple queue runs happening for the same logic.  At a minimum it required an additional DB call after each resource was translated to see if all resources were in the correct status compared to this durable function fan out -> fan in strategy.